src/PostgREST/Types.hs

```diff
@@ -163,7 +163,13 @@ operators = M.fromList [
   ("is", "IS"),
   ("@@", "@@"),
   ("@>", "@>"),
-  ("<@", "<@")]
+  ("<@", "<@"),
+  ("&&", "&&"),
```

Comment:
this is a special char in urls, it should be something else.
like notice we have `("gte", ">=")`, i think things like `<<, && ...` need to be changed to something else 
I am not sure to what but after the tests pass, let's discuss the options then you just change the tuples and the tests